paddlenlp/datasets/bq_corpus.py

```diff
@@ -0,0 +1,61 @@
+import collections
+import json
+import os
+
+from paddle.dataset.common import md5file
+from paddle.utils.download import get_path_from_url
+from paddlenlp.utils.env import DATA_HOME
+from . import DatasetBuilder
+
+__all__ = ['bq_corpus']
+
+
+class bq_corpus(DatasetBuilder):
+    """
+    bq_corpus
```

Comment:
It would be terrific if you provide more information, such as task and auther, for your dataset.